docs/index.md

```diff
@@ -104,7 +108,6 @@ Start GlideinWMS Pilot Submissions
 
 To begin running [GlideinWMS](http://www.uscms.org/SoftwareComputing/Grid/WMS/glideinWMS/) pilot jobs at your site, e-mail <osg-gfactory-support@physics.ucsd.edu> and tell them that you want to start accepting Glideins. Please provide them with the following information:
 
--   The type of CE (HTCondor-CE or the now-unsupported GRAM-CE)
 -   The fully qualified hostname of the CE
 -   Resource/WLCG name
 -   OS major version of your worker nodes — EL 6, EL 7, or a mix of both?
```

Comment:
Question mark not needed. Also, I can't add a comment below for some reason, but you should "Do you accept multicore jobs?" to "Support for multicore jobs" to match phrasing